ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -72,6 +76,9 @@ const EthOverview = ({ className }) => {
   });
   const defaultSwapsToken = useSelector(getSwapsDefaultToken);
 
+  const treatSecondaryCurrencyAsPrimary =
```

Comment:
is this logic duplicated anywhere? Are there any cases where we wouldn't swap these types if there isn't a conversion rate and the primary currency is not the native currency?

Reply:
This precise logic doesn't appear to be duplicated anywhere. And as far as I know we would never want to show the "fiat" (or userPreferred currency) when we don't have an accurate conversionRate.

Reply:
@brad-decker the pattern isn't as clear as I would like it to be and, as I said in a separate thread, I have a dormant larger refactor. But basically we're not showing the user preferred currency if we don't have a valid conversionRate and this placeholder piece of logic allows us to determine whether or not to sub in the native currency in this scenario.